localstack-core/localstack/runtime/server/twisted.py

```diff
@@ -33,8 +35,13 @@ def register(
 
         # add endpoint for each host/port combination
         for host_and_port in listen:
-            # TODO: interface = host?
-            endpoint = endpoints.TCP4ServerEndpoint(reactor, host_and_port.port)
+            if is_ipv6_address(host_and_port.host):
+                endpoint = endpoints.TCP6ServerEndpoint(
+                    reactor, host_and_port.port, interface=host_and_port.host
+                )
+            else:
+                # TODO: interface = host?
```

Comment:
I preserved the existing behavior for IPv4 hosts here, even though it looks like a bug.